tests/components/dynalite/test_switch.py

```diff
@@ -22,7 +24,7 @@ async def test_switch_setup(hass, mock_device):
     """Test a successful setup."""
     await create_entity_from_device(hass, mock_device)
     entity_state = hass.states.get("switch.name")
-    assert entity_state.attributes["friendly_name"] == mock_device.name
+    assert entity_state.attributes[ATTR_FRIENDLY_NAME] == mock_device.name
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)